resources/views/components/modals/2fa-password-confirmation.blade.php

```diff
@@ -0,0 +1,75 @@
+@props([
+    'actionMethod',
+    'closeMethod',
+    'title',
+    'description',
+    'image' => '/images/auth/confirm-password.svg',
```

Comment:
does the project need to supply an image to use? if so, maybe useful for the readme? 🤔

Reply:
it does indeed, I'll add it to the readme